contentcuration/contentcuration/serializers.py

```diff
@@ -238,13 +240,53 @@ class Meta:
     model = ContentTag
     fields = ('tag_name', 'channel', 'id')
 
+
+class AssessmentListSerializer(serializers.ListSerializer):
+    def update(self, instance, validated_data):
+        ret = []
+        file_mapping = {}
+
+        with transaction.atomic():
```

Comment:
This is getting too nested for me. One way to de-nest this is to use `transaction.atomic` as a decorator: https://docs.djangoproject.com/en/1.10/topics/db/transactions/#django.db.transaction.atomic